src/common/utils/backup.js

```diff
@@ -57,3 +65,112 @@ export function ensureBackupFullPath() {
 
   ensureBackupTodayPath()
 }
+
+export function deleteOldBackups(strategy, amount) {
+  const files = sortFileNamesByDate(backupFiles(BACKUP_BASE_PATH))
+
+  switch (strategy) {
+    case 'days': {
+      const anchorDate = nDaysAgo(amount)
+      const filesToDelete = files.filter((file) => !fileIsSoonerThan(anchorDate, file))
+      console.warn('`Removing old backups: ${filesToRemove}')
+      filesToDelete.forEach((file) => {
+        fs.unlinkSync(path.join(BACKUP_BASE_PATH, file))
```

Comment:
there's actually a function in electron that will send these files to the trash instead of deleting them

`shell.moveItemToTrash`

(in the newest version of electron it's: https://www.electronjs.org/docs/api/shell#shelltrashitempath)